### ISSUE TITLE
ascent.mk not correct when vtk-h is turned off

### ISSUE THREAD

**Issue by cyrush** [Alpine-DAV/ascent#305]

**Comment by cyrush** [comment]
```
/home/vsts/work/1/s
g++ -I /home/vsts/work/1/s/uberenv_libs/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.5.0/conduit-master-aeq2tlvdvsqwgdke2v4sdiywrtegl6la/include/conduit   -I/home/vsts/work/1/s/uberenv_libs/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.5.0/hdf5-1.8.21-xguzlrjck6livhpvqfzq7xjxu7dvc2n4/include -I /home/vsts/work/1/s/install/include/ascent  ascent_render_example.cpp -Wl,-rpath,/home/vsts/work/1/s/install/lib -Wl,-rpath,/home/vsts/work/1/s/uberenv_libs/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.5.0/conduit-master-aeq2tlvdvsqwgdke2v4sdiywrtegl6la/lib   -Wl,-rpath,/home/vsts/work/1/s/uberenv_libs/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.5.0/hdf5-1.8.21-xguzlrjck6livhpvqfzq7xjxu7dvc2n4/lib      -L /home/vsts/work/1/s/install/lib -lascent -lascent_flow -lrover -L /lib -lvtkh_rendering -lvtkh_filters -lvtkh_core -lvtkh_lodepng -lvtkh_utils -L /lib -lvtkm_cont- -lvtkm_rendering- -L /home/vsts/work/1/s/uberenv_libs/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.5.0/conduit-master-aeq2tlvdvsqwgdke2v4sdiywrtegl6la/lib -lconduit_blueprint -lconduit_relay -lconduit   -L /home/vsts/work/1/s/uberenv_libs/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.5.0/hdf5-1.8.21-xguzlrjck6livhpvqfzq7xjxu7dvc2n4/lib -lhdf5  -ldl -lrt -pthread    -fopenmp -o ascent_render_example
/usr/bin/ld: cannot find -lrover
/usr/bin/ld: cannot find -lvtkh_rendering
/usr/bin/ld: cannot find -lvtkh_filters
/usr/bin/ld: cannot find -lvtkh_core
/usr/bin/ld: cannot find -lvtkh_lodepng
/usr/bin/ld: cannot find -lvtkh_utils
/usr/bin/ld: cannot find -lvtkm_cont-
/usr/bin/ld: cannot find -lvtkm_rendering-
collect2: error: ld returned 1 exit status
Makefile:75: recipe for target 'main' failed
make: *** [main] Error 1
```

**Comment by cyrush** [comment]
resolved in #346
